.github/workflows/check-links.yml

```
@@ -0,0 +1,21 @@
+name: Check links in Markdown files
+on:
+  schedule:
+    - cron: 0 0 * * 1 # midnight every Monday
+  push:
+    branches: [gh-pages]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  check-links:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: gaurav-nelson/github-action-markdown-link-check@v1
+        with:
+          use-quiet-mode: yes
+          use-verbose-mode: yes
+          folder-path: "_episodes, _extras, _includes"
+          file-path: "./README.md, ./LICENSE.md, ./CODE_OF_CONDUCT.md, CONTRIBUTING.md, reference.md, setup.md, index.md"
```

```suggestion
          file-path: "./_config.yml, ./README.md, ./LICENSE.md, ./CODE_OF_CONDUCT.md, CONTRIBUTING.md, reference.md, setup.md, index.md"
```

At least one broken Imperial link in _config.yml, likely others too

---

Since this is a markdown link checker, I don't believe it will be able to check a yaml file, I'll make the commit just the check. Where/how are those links used in the course notes?

---

Ahh I see, fair enough! Let's get this merged then as it's clearly working on the markdown files.

The grad school link that's failing is the link behind the icon on the left-hand side of the toolbar throughout the course. Probably never been clicked by anyone ever, but still haha.